bndtools.core/src/bndtools/central/EclipseWorkspaceRepository.java

```diff
@@ -79,4 +80,15 @@ protected BiFunction<ResourceBuilder, File, ResourceBuilder> indexer(IProject pr
 	public String toString() {
 		return NAME;
 	}
+
+	@Override
+	public boolean refresh() throws Exception {
+		initialize(null);
+		return true;
+	}
+
+	@Override
+	public File getRoot() throws Exception {
+		return null;
```

Comment:
@pkriens also here I did not know what else to return other than null. Is there a "root" **File** for EclipseWorkspaceRepository ?

Reply:
I recall this was ok. Remains of the old days.